src/vs/base/common/async.ts

```diff
@@ -34,7 +34,7 @@ export function createCancelablePromise<T>(callback: (token: CancellationToken)
 		});
 	});
 
-	return new class implements CancelablePromise<T> {
+	return <CancelablePromise<T>>new class implements Omit<CancelablePromise<T>, typeof Symbol.toStringTag> {
```

Comment:
Not sure if there's a better way to express this. The problem is that this file needs to work with our ie11 monaco build as well, which doesn't include `Symbol`